lib/controllers/journal.js

```diff
@@ -100,8 +100,12 @@ function authorizeFromData(access) {
 }
 
 // set whether the user is a clinician or not in the log data
+// help desk roles are treated as clinicians for the purpose of notes
 function setRole(req, res, next) {
     req.data.role = req.user.role;
+    if (req.data.role === "helpDesk") {
+      req.data.role = "clinician";
+    }
```

Comment:
Let's talk about this. I figure you did it for some good reason, and I'd love to learn about it.

Reply:
Does this affect the if statement later in setData for the journalEntry where we have

```javascript
    // It just empties the data object
    if (this.role === "helpDesk") {
      data = { date: data.date };
    }
```

Reply:
Good point let me trace it